src/components/NotesPage/NotesPageSagas.js

```diff
@@ -0,0 +1,32 @@
+import { all, call, put, takeEvery } from 'redux-saga/dist/redux-saga-effects-npm-proxy.cjs';
+
+import {
+  CHANGE_HELP_VIEW_ACTION,
+  changeHelpViewAction,
+  changeSelectedNoteAction
+} from 'components/NotesPage/NotesPageActions';
+import thoughtStorage from 'storage/thought-storage';
+import { APP_INIT_SUCCESS } from 'components/App/AppActions';
+
+const HELP_VIEWED_KEY = 'controls_help:viewed';
+
+function* saveViewState({ data }) {
+  yield call([localStorage, localStorage.setItem], HELP_VIEWED_KEY, data);
+}
+
+function* readViewState() {
+  const value = Boolean(yield call([localStorage, localStorage.getItem], HELP_VIEWED_KEY));
+  yield put(changeHelpViewAction(value));
+}
+
+function* setRootNote() {
+  yield put(changeSelectedNoteAction(thoughtStorage.getRoot()));
+}
+
+export function* notesInit() {
+  yield all([
+    readViewState(),
+    takeEvery(APP_INIT_SUCCESS, setRootNote),
+    takeEvery(CHANGE_HELP_VIEW_ACTION, saveViewState)
```

Comment:
`takeEvery(USER_CLOSED_HELP, rememberUserSawHelp)` would be more human-readable.

Reply:
This action using not only when user close a help. It also update it on app load, to save value to storage.